examples/11_advanced_parametric_study.py

```diff
@@ -98,9 +98,9 @@ class provides data management features that make the work easier. Also, the
 # Specify heater temperatures. Valid values are 20 - 500 C.
 initial_heater_temps = [80]
 # Restrict the permutations within a range of energy densities
-# For single bead, the energy density is laser power / (laser scan speed * layer thickness).
-min_energy_density = 2e6
-max_energy_density = 8e6
+# For single bead, the pv ratio is laser power / laser scan speed.
```

Comment:
@julieatansys None of our documentation uses "laser velocity" or "v". Up until now, we have always used "laser scan speed". Do we introduce "laser velocity" here or change "PV" to "PS", short for "laser power versus laser scan speed"?

Reply:
**UPDATE**! 
In the AM Process team stand-up meeting, we decided to go with:

- **Laser Scan Speed** is still OK for the parameter name itself
- **P/V Ratio** for the constraint, with uppercase V for velocity
- P-V space or P-V map or P-V plot when referring to the results diagram, such as: "Laser Power and Laser Scan Speed can be plotted to compare all the process parameter combinations in the P-V space."